### PR TITLE
Generate node sdk for Microsoft.DevTestLab provider.

### PR DESCRIPTION
**Pull request by shichaoh**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@shichaoh__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (shichaoh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: lib/services/devTestLabs/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+```javascript
+var DevTestLabsClient = require('azure-arm-devtestlabs');
+var client = new DevTestLabsClient(credentials, 'your-subscription-id');
+```
+
+## Get properties of a lab
+
+```javascript
+client.labOperations.getResource(resourceGroupName, labName, null, function (err, result, request, response) {
```
Comment (amarzavery) [inline lib/services/devTestLabs/README.md]: You don't need to specify null for options. The generated code takes care of it.

**Comment by amarzavery** [comment]
@shichaoh - have you added tests? The onboarding doc provides you information about how to write tests and how to record them etc. https://github.com/Azure/azure-sdk-for-node/tree/master/Documentation

Take a look at the storage management tests for examples.

**Comment by shichaoh** [comment]
@amarzavery yes, I added the test and recording. If you see the CI job details, https://travis-ci.org/Azure/azure-sdk-for-node/jobs/131818527, it has a log "DevTestLabs Get Lab should get integration lab successfully:   ․ DevTestLabs Get Lab should get integration lab". We only added 1 test case for now to make sure end to end is working, we will work on more tests in later commits.

File: lib/services/devTestLabs/README.md (hunk 1 above)
Comment (shichaoh) [inline lib/services/devTestLabs/README.md]: thanks! updated it.

**Pull request merged**